hedera-node/hedera-file-service-impl/build.gradle.kts

```diff
@@ -28,14 +28,17 @@ configurations.all {
 }
 
 dependencies {
+  implementation(project(mapOf("path" to ":hedera-node:hedera-config")))
+  implementation(project(mapOf("path" to ":hedera-node:hedera-config")))
```

Comment:
duplicated line.

Reply:
+1
and please do not use mapOf(...)

You can simply write:
implementation(project(":hedera-node:hedera-config"))